src/containers/EditSubjectFrontpage/components/SubjectpageArticles.tsx

```diff
@@ -44,19 +48,39 @@ const getSubject = (elementId: string) => {
 };
 
 const SubjectpageArticles = ({ editorsChoices, elementId, fieldName }: Props) => {
+  const { query, delayedQuery, setQuery, page, setPage } = usePaginatedQuery();
   const { i18n, t } = useTranslation();
   const [resources, setResources] = useState<(IArticle | ILearningPathV2)[]>(editorsChoices);
   const { setFieldTouched } = useFormikContext();
   const [fieldInputProps] = useField<(IArticle | ILearningPathV2)[]>(fieldName);
   const subjectId = getSubject(elementId);
 
+  const searchQuery = useMoviesQuerySearch(
+    {
+      page,
+      subjects: subjectId,
+      sort: "-relevance",
+      "page-size": 10,
+      query: delayedQuery,
+      "resource-types":
+        "urn:resourcetype:documentary,urn:resourcetype:featureFilm,urn:resourcetype:series,urn:resourcetype:shortFilm",
```

Comment:
Dette er filter som kun trengs for ndla-film. I vanlig artikkel-søk på fagforsider skal ikkje disse være med. Når det er sagt så brukes ikkje dette nokon plass i dag.

Reply:
Tidligere har ikkje dette hatt effekt siden parameter var feil.